website/docs/r/cloudfront_distribution.html.markdown

```diff
@@ -45,11 +45,8 @@ locals {
 resource "aws_cloudfront_distribution" "s3_distribution" {
   origin {
     domain_name = aws_s3_bucket.b.bucket_regional_domain_name
+    origin_access_control_id = aws_cloudfront_origin_access_control.default.id
     origin_id   = locals.s3_origin_id
-
-    s3_origin_config {
-      origin_access_identity = "origin-access-identity/cloudfront/ABCDEFG1234567"
-    }
   }
```

Comment:
Figured it was better to lead off the example to use the new `origin_access_control_id` method with an origin access control.